js_modules/dagit/packages/core/src/metadata/MetadataEntry.tsx

```diff
@@ -160,7 +165,7 @@ export const MetadataEntry: React.FC<{
     case 'EventTableMetadataEntry':
       return <>TableMetadata</>;
```

Comment:
Not changed in this PR, but this seems a bit odd?

Reply:
I changed this to `return null`-- `TableMetadata` here is a placeholder value because `TableMetadata` is not used by anything yet and I haven't experimented with details of rendering. `dagster-pandera` will use it soon though.